kafka_consumer/tests/test_integration.py

```diff
@@ -64,7 +64,7 @@ def test_no_partitions(aggregator, check, kafka_instance, dd_run_check):
 @pytest.mark.parametrize(
     'is_enabled, metric_count, topic_tags',
     [
-        pytest.param(True, 4, ['topic:marvel', 'topic:dc'], id="Enabled"),
+        pytest.param(True, 6, ['topic:marvel', 'topic:dc'], id="Enabled"),
```

Comment:
why do we have to bump this one if we do not collect metrics for the `unconsumed_topic`?

Reply:
ohhh got it! we do collect them in this test
 
sorry

Reply:
No prob! It can be a bit misleading that the parameter is `metric_count`, I can change it `broker_offset_metric_count` to be more clear.

Reply:
https://github.com/DataDog/integrations-core/pull/15098/commits/db6ce23a8e87b3107125590f77604a53e18004f7